stdlib/public/core/StringComparison.swift

```diff
@@ -283,7 +279,7 @@ internal func _lexicographicalCompare(
 ) -> _Ordering {
   return lhs < rhs ? .less : (lhs > rhs ? .greater : .equal)
 }
-// @inline(never) // TODO: temporary, to inspect code quality
+@inline(__always)
```

Comment:
Why do you think this should always be inlined?

Reply:
There are other overloads of _lexicographicalCompare that were marked that way, so I assumed that's what you had there before you made it @inline(never) with a TODO.